project/src/main/java/com/google/sps/servlets/SearchDataServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ListMultimap;
+import com.google.sps.utilities.CommonUtils;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles search queries and addition of event keywords to index. */
+@WebServlet("/search-data")
+public class SearchDataServlet extends HttpServlet {
+  private ListMultimap<String, String> keywordToEventIds = ArrayListMultimap.create();
+  private static final String KEYWORD = "keyword";
+  private static final String EVENT_ID = "event-id";
+  private static final String DESCRIPTION = "description";
+  
+  /**
+   * Return search results for the keyword specified in the request.
+   *
+   * @param request servlet request
```

Comment:
I feel like the @ part of this javadoc (not the first line) is kinda redundant because this method signature is both self-explanatory and a convention in Java already.

If this were a public-facing API I think it would make more sense to have a full doc like this, but in our case I feel like it kinda clutters the code

Reply:
I think the reason I am keeping it is the style guide requires javadoc for public methods to follow have @param and @return. Yes, it could possibly be cluttering the code but it can help distinguish the servlet methods from helper methods.